### PR TITLE
add balance view completed

### LINKED ISSUE #59: Add Add balance screen

### PR DESCRIPTION
**Pull request by pabelnl**
Closes #59

### CODE REVIEW

File: banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            return
+        } else {
+            let alertView = bananoAlertView(title: "Success:", message: "Your Address has been copied to the clipboard.")
+            present(alertView, animated: false, completion: nil)
+            UIPasteboard.general.string = addressLabel.text
+        }
+    }
+    
+    @IBAction func exchange1ButtonPressed(_ sender: Any) {
+        webView.isHidden = false
```
Comment (luyzdeleon) [inline banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift]: This code is duplicated, create a method and call the method.

File: banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        if webView.isHidden {
+            self.dismiss(animated: false, completion: nil)
+        }else {
+            webView.isHidden = true
+            backButton.layer.borderWidth = 0
+        }
+    }
+    
+    @IBAction func copyButtonPressed(_ sender: Any) {
+        let showError = {
```
Comment (luyzdeleon) [inline banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift]: I feel by now we should create a util function as this same code is duplicated in multiple places of the app.

Comment (pabelnl) [reply banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift]: That should be another ticket.

File: banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift (hunk 1 above)
Comment (pabelnl) [reply banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift]: what code? isHidden?
creating a whole method just to hide an element is unnecessary

Comment (luyzdeleon) [reply banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift]: The whole implementation of `exchange1ButtonPressed ` is the same as `exchange2ButtonPressed `, create a single method that takes the necessary parameters for what it's different between the 2.

File: banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift (hunk 2 above)
Comment (luyzdeleon) [reply banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift]: Then leave a TO-DO note and create the issue.

File: banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift (hunk 1 above)
Comment (pabelnl) [reply banano-quest/banano-quest/ViewControllers/AddBalanceViewController.swift]: Got it

**Pull request merged**